PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -72,6 +73,8 @@ end
 local function CreateCamera()
 	local this = {}
 
+	this.cframe = workspace.CurrentCamera and workspace.CurrentCamera.CoordinateFrame or CFrame.new()
```

Comment:
what do we gain by abstracting cframe? It seems like it just adds a layer of direction that could be broken, and I'm not sure if there are any tangible performance benefits.